addons/account_debit_note/views/account_move_view.xml

```diff
@@ -25,4 +18,19 @@
             </field>
         </field>
     </record>
+
+    <record model="ir.actions.server" id="action_move_debit_note">
+        <field name="name">Debit Note</field>
+        <!-- <field name="sequence">4</field> -->
```

Comment:
not sure if that sequence can play a role in the order you see it in the menu... (it could be below credit note)

Reply:
I was trying to play with it but I don't think the order matters that much